src/sentry/utils/pytest.py

```diff
@@ -113,8 +112,10 @@ def pytest_configure(config):
     patcher = mock.patch('socket.getfqdn', return_value='localhost')
     patcher.start()
 
-    from sentry.runner.initializer import initialize_receivers, fix_south, bind_cache_to_option_store
+    from sentry.runner.initializer import (
+        bootstrap_options, initialize_receivers, fix_south, bind_cache_to_option_store)
 
+    bootstrap_options(settings)
```

Comment:
This makes test more like real life, since this now moves options from SENTRY_OPTIONS and shoves them into `settings.*` vars.